### PR TITLE
Fixed conda environment error and 'num_samples=0' error

### PR DESCRIPTION
**Pull request by Dutra-Apex**
When trying to create a conda venv, I kept getting errors about the libraries not being found. To fix this I removed the build string from the environment.yml file. Code ran fine despite the change.

The second error was 'num_samples=0', and it happened when trying to run 'train.py' while passing a custom image and mask directory. The error was fixed by changing how the image_path and mask_path are created.

Hope this is helpful to anyone that might get the same problems.

### CODE REVIEW

**Comment by xinzihao** [comment]
D:\anaconda\envs\torch\python.exe D:\pythonProject12\workplace\AOT-GAN-for-Inpainting-master\AOT-GAN-for-Inpainting-master\src\train.py 
[**] create folder ../experiments\aotgan_places2_pconv256
D:\anaconda\envs\torch\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and will be removed in 0.15, please use 'weights' instead.
  warnings.warn(
D:\anaconda\envs\torch\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and will be removed in 0.15. The current behavior is equivalent to passing `weights=VGG19_Weights.IMAGENET1K_V1`. You can also use `weights=VGG19_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
  0%|          | 0/10 [00:00<?, ?it/s] @Dutra-Apex   May I ask how to solve this problem，thanks.
